src/bun.js/module_loader.zig

```diff
@@ -2277,6 +2327,26 @@ pub const ModuleLoader = struct {
             const specifier_utf8 = specifier.toUTF8(bun.default_allocator);
             defer specifier_utf8.deinit();
             if (graph.files.get(specifier_utf8.slice())) |file| {
+                if (file.loader == .sqlite or file.loader == .sqlite_embedded) {
+                    const code =
+                        \\/* Generated code */
+                        \\import {Database} from 'bun:sqlite';
+                        \\import {readFileSync} from 'node:fs';
+                        \\export const db = new Database(readFileSync(import.meta.path));
```

Comment:
Is this suppose to be the same for embeded and non-embeded?

Reply:
yeah, readFileSync will do the right thing